File: CarsMatterApp/app/src/main/java/com/andrey/carsmatter/models/Car.java
package com.andrey.carsmatter.models;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class Car {
    @SerializedName("id")
    @Expose
    public String Id;

    @SerializedName("carName")
    @Expose
    public String CarName;

    @SerializedName("lowPrice")
    @Expose
    public float LowPrice;

    @SerializedName("highPrice")
    @Expose
    public float HighPrice;

    @SerializedName("manufactureStartDate")
    @Expose
    public String ManufactureStartDate;

    @SerializedName("manufactureEndDate")
    @Expose
    public String ManufactureEndDate;

    @SerializedName("bodyType")
    @Expose
    public String BodyType;

    @SerializedName("avitoUri")
    @Expose
    public String AvitoUri;

    @SerializedName("base64CarImage")
    @Expose
    public String Base64CarImage;
}

File: CarsMatter.Infrastructure/Models/MsSQL/Car.cs
using Newtonsoft.Json;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations.Schema;

namespace CarsMatter.Infrastructure.Models.MsSQL
{
    public class Car
    {
        public string Id { get; set; }

        public string CarName { get; set; }

        [JsonIgnore]
        public string CarImagePath { get; set; }

        [JsonIgnore]
        public string HttpPath { get; set; }

        [NotMapped]
        public string Base64CarImage { get; set; }

        public int LowPrice { get; set; }

        public int HighPrice { get; set; }

        public string ManufactureStartDate { get; set; }

        public string ManufactureEndDate { get; set; }

        public string AvitoUri { get; set; }
        
        public string BodyType { get; set; }

        public string BrandModelId { get; set; }

        [JsonIgnore]
        [ForeignKey("BrandModelId")]
        public BrandModel BrandModel { get; set; }

        [JsonIgnore]
        public ICollection<UserCar> UsersCars { get; set; }

        public Car()
        {
            this.UsersCars = new List<UserCar>();
        }
    }
}
